modules/services/eks/locals.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  api_enabled_clusters = [
+    for cluster in data.aws_eks_cluster.clusters :
+    cluster if contains(["API", "API_AND_CONFIG_MAP"], cluster.access_config[0].authentication_mode) && (var.onboard_all_clusters || contains(var.clusters, cluster.name))
+  ]
+
+  sysdig_cidrs = ["54.218.164.215/32", "54.244.190.180/32", "44.232.85.27/32"]
```

Comment:
Good call! I can convert it to a variable so that we can inject it dynamically. It's only used in the check below:

```tf
contains(cluster.vpc_config[0].public_access_cidrs, local.sysdig_cidrs[0]))
```

I just thought it'd be good to make sure that the cluster has our IP addresses included in allowlist before proceeding with onboarding. What do you think - do we need that check?